client/extensions/woocommerce/app/order/order-fulfillment/index.js

```diff
@@ -23,6 +23,7 @@ import {
 	getCountriesData,
 	getLabels,
 	isLabelDataFetchError,
+	USPS_COUNTRIES,
```

Comment:
From what I understand, constants like this should live in a separate file called `constants.js` that's in the same directory as the `selectors.js` file. For example, `SITE_REQUEST_FIELDS` in [`wp-calypso/client/state/sites/constants.js`](https://github.com/Automattic/wp-calypso/blob/2e6ea789d57156f1f6d2f5d75eba9a177ac6ccb8/client/state/sites/constants.js).

Reply:
Moved in https://github.com/Automattic/wp-calypso/pull/24976/commits/81d3930548446151e702c20a8695553743c4dc20